app/views/home/index.html.erb

```diff
@@ -1,3 +1,113 @@
-<div class="container-fluid" ng-app="receta">
-A-Pc
-</div>
+<!-- Navigation -->
+<nav class="navbar navbar-default" role="navigation">
+    <div class="container">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="/">
+                A-Pc <small>by A-Maze</small>
+            </a>
+        </div>
+    </div>
+</nav>
+
+<div class="container top-margin">
+    <div class="col-md-12 shadow bottom-margin">
```

Comment:
These columns should be generated in a for loop to make proper use of angularjs.